src/common/cockpitpipetransport.c

```diff
@@ -107,7 +107,7 @@ on_pipe_close (CockpitPipe *pipe,
   if (cockpit_pipe_get_pid (pipe, NULL))
     {
       is_cockpit = g_str_equal (self->name, "cockpit-bridge") ||
-                   g_str_equal (self->name, "cockpit-session");
+                   g_str_has_suffix (self->name, "/cockpit-session");
```

Comment:
Should we relax the test also for `/cockpit-bridge`?

Reply:
Unlike cockpit-session (which is in libexec), cockpit-bridge is usually called from `$PATH`. It could test for equal to `cockpit-bridge` or having a `/cockpit-bridge` suffix, but the latter doesn't happen in practice. I pondered changing it as well, but then rather didn't, as that part isn't broken and I wanted to keep the change targetted and minimal.

Reply:
Makes sense.